discovery-provider/src/api/v1/tracks.py

```diff
@@ -560,6 +561,51 @@ def get(self, version):
         return success_response(trending_tracks)
 
 
+@ns.route(
+    "/trending/underground",
+    defaults={
+        "version": DEFAULT_TRENDING_VERSIONS[TrendingType.UNDERGROUND_TRACKS].name
+    },
+    strict_slashes=False,
+    doc={
+        "get": {
+            "id": """Get Underground Trending Tracks""",
+            "description": """Gets the top 100 trending underground tracks on Audius""",
+        }
+    },
+)
+@ns.route(
+    "/trending/underground/<string:version>",
```

Comment:
do we also want this non-full route for `/trending/underground/<string:version>`?

Reply:
probably ok not too. the story of why we have this is when we update the trending algo, we want to make sure that while services waterfall updates the client sees a consistent view of trending

Reply:
you already did it though, so no need to undo!